.github/workflows/nightly.yml

```diff
@@ -97,9 +97,9 @@ jobs:
     if: always()
     needs: [beta, release, cargo-udeps]
     steps:
-      - uses: slackapi/slack-github-action@v1.19.0
       - uses: technote-space/workflow-conclusion-action@v3
       - name: Post to slack
```

Comment:
```suggestion

      - name: Post to slack
```

Reply:
actually it looks like this workflow doesnt have empty lines between steps so perhaps ignore this..